Kitodo-DataManagement/src/main/java/org/kitodo/data/elasticsearch/search/Searcher.java

```diff
@@ -220,26 +206,24 @@ public JSONObject findDocument(String query, String sort) throws DataException {
      *            as String
      * @return list of JSON objects
      */
-    public List<JSONObject> findDocuments(String query, String sort, Integer offset, Integer size)
+    public List<JsonObject> findDocuments(String query, String sort, Integer offset, Integer size)
             throws DataException {
         SearchRestClient restClient = initiateRestClient();
-        List<JSONObject> searchResults = new ArrayList<>();
-        JSONParser parser = new JSONParser();
+        List<JsonObject> searchResults = new ArrayList<>();
 
         String response = restClient.getDocument(query, sort, offset, size);
-        try {
-            JSONObject jsonObject = (JSONObject) parser.parse(response);
-            if (jsonObject.containsKey("hits")) {
-                JSONObject hits = (JSONObject) jsonObject.get("hits");
-                JSONArray inHits = (JSONArray) hits.get("hits");
-                if (!inHits.isEmpty()) {
-                    searchResults.addAll(inHits);
+        JsonReader jsonReader = Json.createReader(new StringReader(response));
```

Comment:
Same here. `JsonReader` is closeable so using a try-with-resources statement for closing the object is maybe a good idea.